api/jwt/jwt.go

```diff
@@ -179,10 +181,12 @@ func (service *Service) generateSignedToken(data *portainer.TokenData, expiresAt
 		Role:                int(data.Role),
 		Scope:               scope,
 		ForceChangePassword: data.ForceChangePassword,
-		StandardClaims: jwt.StandardClaims{
-			ExpiresAt: expiresAt,
-			IssuedAt:  time.Now().Unix(),
-		},
+	}
+
+	if !expiresAt.IsZero() {
+		cl.RegisteredClaims = jwt.RegisteredClaims{
+			ExpiresAt: jwt.NewNumericDate(expiresAt),
+		}
```

Comment:
should we add `issuedAt`? it's not used anywhere but it was on the claims

Reply:
I was trying to match existing functionality, but it is a reasonable idea.

Reply:
actually that's why I asked. we had it before

Reply:
I need to double check as I've forgotten exactly but I think the default is `time(now)` so you don't need to explicitly set it